source/scale/scale-to-93000-users.rst

```diff
@@ -0,0 +1,60 @@
+Scale Mattermost up to 93000 users
+==================================
+
+.. include:: ../_static/badges/ent-selfhosted.rst
+  :start-after: :nosearch:
+
+This page describes the Mattermost reference architecture designed for the load of up to 93000 concurrent users. Unsure which reference architecture to use? See the :doc:`scaling for enterprise </scale/scaling-for-enterprise>` documentation for details.
+
+- **High Availability**: Required
+- **Database Configuration**: writer, multiple readers
+
+.. note::
+    Usage of CPU, RAM, and storage space can vary significantly based on user behavior. These hardware recommendations are based on traditional deployments and may grow or shrink depending on how active your users are.
+
+Requirements
+------------
+
++------------------------+-----------+----------------+-------------------+
+| **Resource Type**      | **Nodes** | **vCPU/        | **AWS Instance**  |
+|                        |           | Memory (GiB)** |                   |
++========================+===========+================+===================+
+| Mattermost Application | 5         | 16/32          | c7i.4xlarge       |
++------------------------+-----------+----------------+-------------------+
+| RDS Writer             | 1         | 16/128         | db.r7g.4xlarge    |
++------------------------+-----------+----------------+-------------------+
+| RDS Reader             | 4         | 16/128         | db.r7g.4xlarge    |
++------------------------+-----------+----------------+-------------------+
+| Elasticsearch Node     | 0         | 4/32           | r6g.xlarge.search |
```

Comment:
Zero?

Reply:
@nab-77 Yes, assuming that Test 4XL.550 is the correct [4xl](https://mattermost.atlassian.net/wiki/spaces/XYZ/pages/2809004081/Ceiling+Tests+V2) test results